examples/virtual_staining/vcp_tutorials/quick_start.py

```diff
@@ -0,0 +1,311 @@
+choose the 'T4' GPU runtime from the "Connect" dropdown menu
+in the upper-right corner of this notebook for faster execution.
+Using a GPU significantly speeds up running model inference, but CPU compute can also be used.
+
+## Setup Local Environment
+
+The commands below assume a Unix-like shell with `wget` installed.
+On Windows, the files can be downloaded manually from the URLs.
+"""
+
+# %%
+# Install VisCy with the optional dependencies for this example
+# See the [repository](https://github.com/mehta-lab/VisCy) for more details
+# !pip install "viscy[metrics,visual]==0.3.0rc2"
+
+# %%
+# Validate installation
+# !viscy --help
+
+# %%
+# Download the example dataset
+# !wget -m -np -nH --cut-dirs=5 -R "index.html*" "https://public.czbiohub.org/comp.micro/viscy/VS_datasets/VSCyto2D/test/a549_hoechst_cellmask_test.zarr/"
+# Download the model checkpoint
+# !wget https://public.czbiohub.org/comp.micro/viscy/VS_models/VSCyto2D/VSCyto2D/epoch=399-step=23200.ckpt
+
+# %% [markdown]
+"""
+# Run Model Inference
+
+The following code will run inference on a single field of view (FOV) of the example dataset.
+This can also be achieved by using the VisCy CLI.
+"""
+
+# %%
+from pathlib import Path
+
+from iohub import open_ome_zarr
+from torchview import draw_graph
+
+from viscy.data.hcs import HCSDataModule
+from viscy.trainer import VisCyTrainer
+from viscy.transforms import NormalizeSampled
+from viscy.translation.engine import FcmaeUNet
+from viscy.translation.predict_writer import HCSPredictionWriter
+
+# %%
+# NOTE: Nothing needs to be changed in this code block for the example to work.
+# If using your own data, please modify the paths below.
+
+# TODO: Set download paths, by default the working directory is used
+root_dir = Path()
+# TODO: modify the path to the input dataset
+input_data_path = root_dir / "a549_hoechst_cellmask_test.zarr"
+# TODO: modify the path to the model checkpoint
+model_ckpt_path = root_dir / "epoch=399-step=23200.ckpt"
+# TODO: modify the path to save the predictions
+output_path = root_dir / "a549_prediction.zarr"
+# TODO: Choose an FOV
+fov = "0/0/0"
+
+
+# %%
+# Configure the data module for loading example images in prediction mode.
+# See API documentation for how to use it with a different dataset.
+# For example, View the documentation for the HCSDataModule class by running:
+# ?HCSDataModule
+
+# %%
+# Setup the data module to use the example dataset
+data_module = HCSDataModule(
+    # Path to HCS or Single-FOV OME-Zarr dataset
+    data_path=input_data_path / fov,
+    # Name of the input phase channel
+    source_channel="Phase3D",
+    # Desired name of the output channels
+    target_channel=["Membrane", "Nuclei"],
+    # Axial input size, 1 for 2D models
+    z_window_size=1,
+    # Batch size
+    # Adjust based on available memory (reduce if seeing OOM errors)
+    batch_size=8,
+    # Number of workers for data loading
+    # Set to 0 for Windows and macOS if running in a notebook,
```

Comment:
In the past we had set this to be 0 given the Windows/Mac issue.

Reply:
Google Colab always runs on Linux so I'm using MP to make it faster.